src/helpers/hotkeysManager.js

```diff
@@ -251,6 +253,26 @@ WebViewer(...)
           core.setCurrentPage(currPageNumber + 1);
         }
       },
+      up: e => {
+        // do not call preventDefault else it will prevent scrolling
+        const scrollViewElement = core.getScrollViewElement();
+        const { scrollTop } = scrollViewElement;
+        const reachedTop = scrollTop === 0;
+        const currPageNumber = core.getCurrentPage();
+        if ((e.key === 'ArrowUp' || e.which === 38) && reachedTop && currPageNumber > 1) {
+          core.setCurrentPage(currPageNumber - 1);
```

Comment:
The `this.pageUp` in `DocumentContainer.js` has this line `this.container.current.scrollTop = scrollHeight - clientHeight;`. What it does is that when the page is zoomed in and the scroll bar reaches the top, we will go to the previous page but keep the scroll position to the bottom.

To remain the same functionality this line should be copied over here as well.

Reply:
addressed